corehq/apps/case_search/xpath_functions/ancestor_functions.py

```diff
@@ -7,6 +7,18 @@
 from corehq.apps.case_search.xpath_functions.utils import confirm_args_count
 from corehq.apps.case_search.const import MAX_RELATED_CASES
 from corehq.apps.es.case_search import CaseSearchES, reverse_index_case_query
+from corehq.toggles import NO_SCROLL_IN_CASE_SEARCH
+
+
+def _should_scroll(context):
+    if not isinstance(context.domain, list):
+        domains = [context.domain]
+    else:
+        domains = context.domain
+    for domain in domains:
+        if NO_SCROLL_IN_CASE_SEARCH.enabled(domain):
+            return False
```

Comment:
I don't think it really matters, and am not even sure of the context where there are multiple domains, but seems safer to make sure that all domains have `NO_SCROLL_IN_CASE_SEARCH` enabled. Just curious what your thoughts were here.